### PR TITLE
:shirt: refactoring

### PR DESCRIPTION
**Pull request by andela-fokosun**
want help reviewing this code 🗡

### CODE REVIEW

File: src/Smelly/Triangle.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    /**
+     * Calculate the area of a triangle
+     *
+     * @return string
+     */
+    public function area()
+    {
+        return 'half base X height';
+    }
+}
```
Comment (andela-araimi) [inline src/Smelly/Triangle.php]: There should be a newline

File: tests/shapes/CalculateAreaTest.php [hunk 2]
```diff
@@ -4,6 +4,7 @@
 use Solid\Smelly\Rectangle;
 use Solid\Smelly\AreaCalculator;
 use PHPUnit\Framework\TestCase;
+use Solid\Smelly\Triangle;
```
Comment (andela-araimi) [inline tests/shapes/CalculateAreaTest.php]: Nice package

File: tests/shapes/CalculateAreaTest.php [hunk 3]
```diff
@@ -38,4 +39,18 @@ public function testSixFor2X3Rectangle()
 
         $this->assertEquals(6, AreaCalculator::calculateRectangleArea($rect));
     }
+
+    /**
+     * Test for calculating the area of a triangle
+     *
+     * @return void
```
Comment (andela-araimi) [inline tests/shapes/CalculateAreaTest.php]: If the function return void, then remove the return keyword instead.

Comment (andela-fokosun) [inline tests/shapes/CalculateAreaTest.php]: Ok thanks.

File: tests/shapes/CalculateAreaTest.php (hunk 2 above)
Comment (andela-fokosun) [inline tests/shapes/CalculateAreaTest.php]: This is great thanks

**Pull request merged**